_tutorials/install-add-fdroid-repo.html

```diff
@@ -33,10 +33,15 @@ <h3 id="qr-code">The first of these is by scanning QR code or clicking link.</h3
             <img src="/img/screenshots/shot_add_fdroid_repo.png" class="img-responsive screenshot" alt="F-Droid 'Add new repository' pop-up"/>
             <figcaption>
                 <ol start="2">
-                    <li>After scanning, this menu will appear:</li>
+                    <li>After step 1, the menu shown to the right will appear:</li>
+                    <li><b>If, after step 1, you don't get <i>exactly</i> what's shown in this screenshot, then abort this method and use the other one.</b></li>
                     <li>Click <code>OK</code>. The NewPipe upstream repository will be added.</li>
-                    <li>Go to F-Droid's search and search for <code>NewPipe</code>. Click the listing for NewPipe.</li>
-                    <li>Scroll down to the versions tab. Expand both of the tabs for the highest numbers at the top. Press <code>INSTALL</code> on the one that says <code class=".fdroid-repo-code-1">Repository: NewPipe upstream repository</code>.</li>
+                    <li>Even after doing all the previous steps correctly, you'll notice a grey "Unverified" below the repo title. Don't worry. Move to the next step.</li>
+                    <li>F-Droid will now automatically refresh all repo indices, including the new one. Let the process finish.</li>
+                    <li>Once that is done, go to your home screen, and then open the app again. You should now see the proper repo title, "NewPipe upstream repository".</li>                    
+                    <li>Go to "Latest" tab and search for <code>NewPipe</code>. Click the listing for NewPipe.</li>
```

Comment:
I don't like the `<code>` usage here and in the rest of the file; I think double quotes are better suited.

Reply:
No problem. I copied that over from the original.

Reply:
Then let's have all these `<code>` usages changed to quotes in another PR.

Reply:
I meant that I would change them. I just wanted to state that they weren't added by me.